VideoLocker/src/main/java/org/edx/mobile/view/MyVideosTabActivity.java

```diff
@@ -221,6 +221,22 @@ public View createTabContent(String tag) {
         }
     }
 
+    public void enableTabChange() {
+        for (int i=0; i<mTabHost.getTabWidget().getTabCount(); i++) {
+            View tab = mTabHost.getTabWidget().getChildTabViewAt(i);
+            tab.setEnabled(true);
+        }
+        mTabHost.setOnTabChangedListener(listener);
+    }
+
+    public void disableTabChange() {
+        mTabHost.setOnTabChangedListener(null);
+        for (int i=0; i<mTabHost.getTabWidget().getTabCount(); i++) {
+            View tab = mTabHost.getTabWidget().getChildTabViewAt(i);
+            tab.setEnabled(false);
+        }
+    }
+
```

Comment:
I will combine these two methods to just one <code>setTabChangeEnabled(boolean enabled)</code>.